docs/reference/commandline/run.md

```diff
@@ -216,7 +216,9 @@ An example of a file passed with `--env-file`
     _TEST_BAR=FOO
     TEST_APP_42=magic
     helloWorld=true
-    # 123qwe=bar <- is not valid
+    # The two following are not valid in the  "posix" sence but will get passed
```

Comment:
The posix comment is technically incorrect. Posix is quite happy to allow those variable names. What it says is that the shell functions and utilities covered within posix must not use variable names such as those (eg, `sh` is defined in posix, and if you wish to be posix compliant, your implementation of `sh` cannot use a variable called `123qwe`).